token/cli/src/main.rs

```diff
@@ -3060,16 +3088,15 @@ fn process_command(
 
             command_sync_native(address, bulk_signers, config)
         }
-        _ => unreachable!(),
     }
 }
 
-fn format_output<T>(command_output: T, command_name: &str, config: &Config) -> String
+fn format_output<T>(command_output: T, command_name: &CommandName, config: &Config) -> String
 where
     T: Serialize + Display + QuietDisplay + VerboseDisplay,
 {
     config.output_format.formatted_string(&CommandOutput {
-        command_name: String::from(command_name),
+        command_name: <String as From<&str>>::from(command_name.into()),
```

Comment:
nit: how about:
```suggestion
        command_name: command_name.to_string()
```
Should be possible according to https://docs.rs/strum/latest/strum/derive.ToString.html